### ISSUE TITLE
Instructions unclear for logging in with google account

### ISSUE THREAD

**Issue by khag7** [vinnymac/PokeNurse#71]
I've seen some of the closed issues already mentioned this, but I'm adding more info...
1. If using a google account with 2FA, go to genereate an app password here:
   https://security.google.com/settings/security/apppasswords
2. For me it still would not let me log in as khag7@gmail.com but when I dropped the @gmail.com part it worked fine. So for username, instructions should state to not include the domain (which is odd because some folks have google accounts that are @theirowndomain.com instead of @gmail.com) so you might want to look into that.

tl;dr: if using google acct, username is without @gmail.com and password is an app password if you use 2FA

**Comment by vinnymac** [comment]
We have unreleased fixes to provide the user with an error message. Gmail domains definitely work, they are all I have ever used to login. Most likely you're receiving an error when using the domain for some reason but currently cannot see the error.

You do not need to remove your `gmail` domain to sign in with 2FA account.

**Comment by JimmyRoX** [comment]
I have 2FA and I had no issue using my mail @gmail.com just now. I'm using the Mac OS X Lastest version.

**Comment by khag7** [comment]
Maybe its a windows thing but if I try logging in with @gmail.com it doesn't work. I have to use just the first part of my email.

Edit:
I should mention i'm using the latest windows RELEASE, not the development repo. I tried installing current dev version and got some errors on install relating to electron. I am more interested in devoting my development time to mapping projects so I didn't feel like screwing around with it any further and will only be using release versions for this software.

**Comment by vinnymac** [comment]
No changes to sign in have been made in the new release. We use pogobuf to authenticate, and I doubt they are doing anything different on windows. I find that interesting, it seems more likely to me that it is your account interfacing with pogobuf or google, than it is anything we are doing. We literally just take your info and pass it to pogobuf. Do other projects that use pogobuf work with `@gmail.com` for you?

**Comment by khag7** [comment]
I uninstalled and reinstalled and it seems to work both ways now. (with and without @gmail.com). That's the expected behavior, since google logins will assume @gmail if its not included. I'm not sure why it wasn't working before, but it is now. Hell, I guess its possible I was mistyping it the entire time, but I don't know. Sorry for the false alarm.
